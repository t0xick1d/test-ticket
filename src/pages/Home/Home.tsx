import React from 'react';

const Home: React.FC = () => <h1 data-testid="home">Home</h1>;

export default Home;
