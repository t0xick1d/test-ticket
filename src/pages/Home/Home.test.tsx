import React from 'react';
import { render, screen } from '@testing-library/react';
import Home from './Home';

test('check Home', () => {
   render(<Home />);
   expect(screen.getByTestId('home')).toBeInTheDocument();
});

test('Home matches snapshot', () => {
   const { asFragment } = render(<Home />);
   expect(asFragment()).toMatchSnapshot();
});
