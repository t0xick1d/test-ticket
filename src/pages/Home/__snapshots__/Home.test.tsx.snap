// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches snapshot 1`] = `
<DocumentFragment>
  <h1
    data-testid="home"
  >
    Home
  </h1>
</DocumentFragment>
`;
